index.js

```diff
@@ -31,30 +48,30 @@ module.exports = function (stream, reducer, opts) {
     binder.onUpdate()
   }
 
-  pull(
-    stream,
-    pauser,
-    pull.drain((item) => {
-      if (item.sync) {
-        sync.set(true)
-      } else {
-        result.push(item)
-      }
-    }, () => {
-      sync.set(true)
-    })
-  )
-
   return result
 
+  /**
+   * Start or resume the stream from the last value using the user supplied function.
+   */
+  function startStream() {
+    pull(
+      getStream(binder.getValue()),
```

Comment:
I think this value should be the last item that came through the stream, rather than the reduced state.

Reply:
True =]. It occurred to me when I hacked the index.js file to bits to try and work out why the unit tests aren't passing.

I've updated the PR to use the last stream value. The unit tests are still failing - will try and work out what the deal is with those later =p.